h/schemas/forms/accounts/change_email.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import colander
+import deform
+
+from h import i18n
+from h.schemas import validators
+from h.schemas.base import CSRFSchema
+from h.schemas.forms.accounts.validators import unique_email
+from h.models.user import (
+    EMAIL_MAX_LENGTH
+)
+
+_ = i18n.TranslationString
+
+
+def _validate_unique_email(*args):
+    unique_email(*args)
+
+
+class ChangeEmailSchema(CSRFSchema):
+
+    email = colander.SchemaNode(
```

Comment:
This duplicates the `email_node` logic that is also used by `RegisterSchema`. I think it makes sense to have a single implementation to ensure that the requirements are kept the same during initial registration and when changing an email later. Additionally it seems to me that this could avoid some duplication in the tests.